privacyidea/api/lib/postpolicy.py

```diff
@@ -262,18 +262,17 @@ def check_tokeninfo(request, response):
     :return: A new modified response
     """
     content = json.loads(response.data)
-    policy_object = g.policy_object
     serial = content.get("detail", {}).get("serial")
-    if serial:
+    tokeninfos_pol = g.policy_object.get_action_values(
```

Comment:
In this case you will always execute ``get_action_values``.
Wouldn't it be wise to only do this if there is actually a serial? e.g. put this in the if block again.

Reply:
Yep, sounds good! Will do.

Reply:
fixed in a1740172ecbba64493d89bb14edbd21e2f7e08f4